src/VisualStudio/Core/Def/Implementation/Remote/WellKnownRemoteHostServices.cs

```diff
@@ -0,0 +1,11 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+namespace Microsoft.VisualStudio.LanguageServices.Remote
+{
+    internal class WellKnownRemoteHostServices
+    {
+        public const string RemoteHostService = "remoteHostService";
```

Comment:
that is convention of service hub of xxx.servicehub.json file. I need xxx as service name.